api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/constraint/EligibilityOverrideValidator.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.dhsc.htbhf.claimant.entity.constraint;
+
+import org.apache.commons.lang3.ObjectUtils;
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EligibilityOverrideValidator implements ConstraintValidator<ValidEligibilityOverride, EligibilityOverride> {
+    @Override
+    public void initialize(ValidEligibilityOverride constraintAnnotation) {
+        //No initialization required
+    }
+
+    @Override
+    public boolean isValid(EligibilityOverride eligibilityOverride, ConstraintValidatorContext context) {
+        if (null == eligibilityOverride) {
```

Comment:
Minor - having the check this way round is unconventional, can we have `eligibilityOverride == null` instead please as it reads better?